development-defaults.ini

```diff
@@ -113,16 +149,8 @@ no_shirt     = 0
 'Squarewave Lanyard'          = 3
 
 
-
 [enums]
 
-[[badge_status]]
```

Comment:
I know adding badge statuses is an AC change, but this is actually something I'd like to use at MAGFest. It would help us figure out if, e.g., a person was refunded because they got a free badge (in which case the status would be "Completed") or if the badge went unused for some reason (a lost badge replacement, special exceptions to our no-refund policy).